### PR TITLE
Make the mv-backendType-* family of attributes live

### PR DESCRIPTION
**Pull request by DmitrySharabin**
It might be I wasn't supposed to take this issue since it's labeled as a good first issue. 🙈 I just thought that it would be nice if we have this feature asap.

### CODE REVIEW

File: src/mavo.js [hunk 1]
```diff
@@ -161,7 +161,9 @@ var _ = self.Mavo = $.Class({
 
 		this.observe({deep: false, attribute: true}, ({attribute}) => {
 			if (attribute.indexOf("mv-") === 0) {
-				let role = attribute?.replace(/^mv-/, "");
+				// We want to observe changes both in a backend (the mv-role attribute)
+				// and its metadata (provided via the mv-role-* family of attributes)
+				let [role, ...rest] = attribute?.replace(/^mv-/, "")?.split("-");
```
Comment (LeaVerou) [inline src/mavo.js:166]: It seems a bit misleading to use a variable that is never used. What about:

```js
let role = attribute?.replace(/^mv-/, "")?.split("-")?.[0];
```

**Review by LeaVerou — COMMENTED** [review]
Thanks for working on this! See (minor) comment (which is only an idea, don't feel obligated to follow that)

File: src/mavo.js (hunk 1 above)
Comment (DmitrySharabin) [reply src/mavo.js]: You are absolutely right! That's better. Fixed. Thank you. 🙏🏻

**Comment by LeaVerou** [comment]
Thanks, merged!

**Pull request merged**